DigitalLearningSolutions.Web/appsettings.Development.json

```diff
@@ -2,7 +2,7 @@
   "ConnectionStrings": {
     "DefaultConnection": "Data Source=localhost;Initial Catalog=mbdbx101;Integrated Security=True;"
   },
-  "CurrentSystemBaseUrl": "https://localhost:44367",
+  "CurrentSystemBaseUrl": "https://localhost:44363",

```

Comment:
This should not be changed, it's pointing to the old code running locally 🙂

Reply:
Thanks, I wondered if I was breaking something there. Reverted in dba5ff9 🙂